rasa/core/domain.py

```diff
@@ -112,9 +112,9 @@ def from_dict(cls, data: Dict) -> "Domain":
         utter_templates = cls.collect_templates(data.get("templates", {}))
         slots = cls.collect_slots(data.get("slots", {}))
         additional_arguments = data.get("config", {})
-        intent_properties = cls.collect_intent_properties(data.get("intents", {}))
+        intent_list = data.get("intents", {})
```

Comment:
Avoid hungarian notation
```suggestion
        intents = data.get("intents", {})
```